Will Heikoff Documentation

```diff
@@ -1,13 +1,34 @@
+Will Heikoff Documentation
+Goal: To get the python script K-function Local, G:\MAX-Filer\Research\Labs-kbuzard-S18\kfunctions\K-function Local, for California R & D Labs running for 9 iterations
+
+6/12/18
+Searched for file which exhibits the zip code with the associated Total and manufacturing employment for the zip code.
+
+6/15/18
+Found zip code file for line 22: ZIPS.shp, using trial and error methods testing different files with the name ‘zips’ within the California folder, plugging the files into the ArcMap file ‘Base’, located within the same folder, to find a file which exhibits the zip code with the associated Total and manufacturing employment for the zip code.
+	Located at G:\MAX-Filer\Research\Labs-kbuzard-S18\CARR\California\Base Data
 Entered ZIPS.shp into line 22
-Found labs data for line 23: Labs.shp
+Found labs data for line 23: Labs.shp, using trial and error methods testing different files with the name ‘zips’ within the California folder, plugging the files into the ArcMap file ‘Base’, located within the same folder, to find a file which exhibits the Labs street address and associated ‘X’ and ‘Y’ cooridinates
+	Located at G:\MAX-Filer\Research\Labs-kbuzard-S18\CARR\California\Base Data
 Entered Labs.shp into line 23
-Entered ‘ZIP’ on Line 26 for variable name
-Entered variable names ‘X’ and ‘Y’ variable names into respective spots on line 47-48
+Entered ‘ZIP’ on Line 26 for variable name, after plugging in the ‘ZIPS.zhp’ file into ArcMap and using the attribute Table to establish what the zip code variable was
+Entered variable names ‘X’ and ‘Y’ variable names into respective spots on line 47-48, after plugging the ‘Labs.shp’ file into ArcMap and using the attribute Table to establish what the Coordinate variables are
 Set simulations to 9 on Line 39
 Put parenthesis around all ‘print’ lines to adjust for the syntax change from Python 2.7 to 3.6
-Set Python Path by going to Tools and then “PYTHONPATH manager”; adding path ‘C:\Program Files (x86)\ArcGIS\Desktop10.5’ 
-  therefore allowing spyder to import Arcpy
-Set Python Path using “PYTHONPATH manager”; adding path ‘G:\MAX-filer\Research\Labs-kbuzard-S18\kfunctions’ therefore 
-  allowing spyder to import kfunctions
-Troubleshooting error arising from "AttributeError: module 'arcpy' has no attribute 'env' " on Line 29
+Set Python Path by going to Tools and then “PYTHONPATH manager”; adding path ‘C:\Program Files (x86)\ArcGIS\Desktop10.5’ therefore allowing spyder to import Arcpy
+Set Python Path using “PYTHONPATH manager”; adding path ‘G:\MAX-filer\Research\Labs-kbuzard-S18\kfunctions’ therefore allowing spyder to import kfunctions
+
+6/20/18
+Troubleshooting issue with arcpy.env; Spyder has the issue “AttributeError: module 'arcpy' has no attribute 'env' “
+
+6/21/18
+Replaced kfunction, K-function Local, and K-function Global with kfunction_scipy, K-function Local_scipy, and K-function_scipy, to help resolve the issue locating the ‘env’ attribute for arcpy
+
+6/22/18
```

Comment:
I really like how this whole section from yesterday is written.

Next step: more specific commit messages. Look through some of my other repositories to get a feel for the types of messages I find useful. For instance, just a little bit of detail about what is contained in the update is more useful for us in the future than just saying it's an update.

For this one, maybe something like "add full file paths to documentation and switch to scipy versions of kfunctions"